command_line/stills_process.py

```diff
@@ -240,17 +241,27 @@ def do_import(filename, load_models=True):
 
     from dxtbx.imageset import ImageSetFactory
 
+    all_experiments = ExperimentList()
     for experiment in experiments:
-        if load_models:
-            experiment.load_models()
+        # Convert from ImageSequence to ImageSet, if needed
         imageset = ImageSetFactory.imageset_from_anyset(experiment.imageset)
-        imageset.set_scan(None)
-        imageset.set_goniometer(None)
-        experiment.imageset = imageset
-        experiment.scan = None
-        experiment.goniometer = None
+        for i in range(len(imageset)):
+            # Preserve original models if they were available (in the case of an image file
+            # they will not be, but in the case of a previously processed experiment list,
+            # then they may be available
+            expt = Experiment(
+                imageset=imageset[i : i + 1],
+                detector=experiment.detector,
+                beam=experiment.beam,
+                scan=experiment.scan,
+                goniometer=experiment.goniometer,
```

Comment:
Since you are expecting  `scan = None` and `goniometer = None` why not enforce that here? Thinking that merging https://github.com/dials/dials/pull/1148 will have an impact on this choice.

Reply:
I was trying to go by the idea that not deleting scans/gonios was desired.  I tested this code with #1148 and the stills indexer and stills integrator were both picked, so I'm happy enough.